.gitignore

```diff
@@ -1,9 +1,5 @@
 /target
 /result
-cfg
```

Comment:
I actually still use that one :stuck_out_tongue_closed_eyes: (adding it to `.git/info/exclude` as I should have from the beginning)